### DISCUSSION TITLE
url bar text improvement

### DISCUSSION THREAD

**Discussion by shalva97** [minbrowser/min#1506] [Completed]
some links are really long, like [this one](https://netdata.cloud/alarms/redirect?agentID=71e3781e-626a-13eb-9cd9-c45444969ff6&host=accer&chart=ipv4.udperrors&family=udp&alarm=1m_ipv4_udp_send_buffer_errors&alarm_unique_id=16119567330&alarm_id=16121949561&alarm_event_id=4505&alarm_when=161351411), and if I click the tab to see what website is open it shows just the end of the link and I need to highlight the text to scroll it or press Home

It would be better if it always showed start of the website

**Comment by PalmerAL** [comment]
Yeah I've seen this too, we should fix it.

**Comment by PalmerAL** [comment]
Fixed in cb6fce4994615ebe043e8b310b18228eef25d46f.
